templates/jws53-openj9-tomcat9-rhel8-basic-s2i.json

```diff
@@ -7,17 +7,17 @@
             "openshift.io/provider-display-name": "Red Hat, Inc.",
             "tags": "tomcat,tomcat9,java,jboss",
             "version": "1.0",
-            "openshift.io/display-name": "JBoss Web Server 5.2 Apache Tomcat 9 OpenJ9 JDK11 (no https)",
+            "openshift.io/display-name": "JBoss Web Server 5.3 Apache Tomcat 9 OpenJ9 JDK11 (no https)",
             "description": "An example JBoss Web Server application. For more information about using this template, see https://github.com/jboss-openshift/application-templates.",
-            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat JBoss Web Server 5.2 Apache Tomcat 9 OpenJ9 JDK11 based application, including a build configuration, and an application deployment configuration.",
+            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat JBoss Web Server 5.3 Apache Tomcat 9 OpenJ9 JDK11 based application, including a build configuration, and an application deployment configuration.",
             "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-web-server/",
             "template.openshift.io/support-url": "https://access.redhat.com"
         },
-        "name": "jws52-openj9-tomcat9-rhel8-basic-s2i"
+        "name": "jws53-openj9-tomcat9-rhel8-basic-s2i"
     },
     "labels": {
-        "template": "jws52-openj9-tomcat9-rhel8-basic-s2i",
-        "jws52openj9": "1.7"
+        "template": "jws53-openj9-tomcat9-rhel8-basic-s2i",
+        "jws53openj9": "1.7"
```

Comment:
@levivic I would recommend to change the label to jws53openj9el8 in all the tempaltes, in the same fashion that we have for the rhel7 ones, and also reset the value to 1.0 since this will be the first release. After that we can cut a new tag and reference the templates with a version jws53openj9el8-v1.0 .

Reply:
Changes applied, thanks for the comments.